### ISSUE TITLE
Create fossrit.community subdomain for foss-profiles

### ISSUE THREAD

**Issue by jwflory** [FOSSRIT/foss-profiles#71]
# Summary

Create a subdomain on `fossrit.community` for the new foss-profiles GitHub Pages site

# Background

Over in FOSSRIT/foss-profiles#69, @kennedy casually whipped together a new Jekyll site that turns our nifty collection of YAML profiles of folks at RIT into an actual web app. Once his PR is merged and the site is published, it would be nice to give it a subdomain to make its web presence a little more official.

If a sub-domain is not assigned, I'm pretty sure it will default to `fossrit.github.io/foss-profiles/`. Which works, but it is a little wieldy.

Also, the intended audience for this web app is internal to RIT, i.e. people who are already involved and active with the FOSS@RIT community. Currently I don't expect us to bring this to conferences or link it all around the web. It's mostly a reference for people who are at RIT to see who folks doing open source at RIT are and a little bit on what they are doing.

# Details

I wanted to open this up for community input on what the subdomain is. I don't want to spend _too_ much time focusing on a very narrow detail, but once we commit to a subdomain, it's important we stick with it for the benefit of long-term persistence.

Here's the options I was thinking of:

* `whois.fossrit.community`
* `directory.fossrit.community`
* `dir.fossrit.community`

I'm particular to `whois.fossrit.community` but open to suggestions.

# Outcome

Dedicated web presence for new foss-profiles website that we can share around with folks at RIT, and possibly round up more profiles for people we are just starting to meet doing open source at RIT

**Comment by jwflory** [comment]
Given the story in #71 where we want to integrate this more closely with the FOSS@RIT website, I don't think a subdomain makes sense. Closing this as `wontfix`, and we can continue discussion in #78 about URLs.
